src/listener/starboard/starRemove.ts

```diff
@@ -0,0 +1,79 @@
+import { Listener } from 'discord-akairo';
+import { guildConfigs } from '../../guild/config/guildConfigs';
+import { MessageReaction } from 'discord.js';
+import { TextChannel } from 'discord.js';
+import { User } from 'discord.js';
+import { Starboard } from '../../structure/starboard/star';
+import { StarModel } from '../../model/star';
+import { MessageEmbed } from 'discord.js';
+
+export default class StarRemoveListener extends Listener {
+	constructor() {
+		super('starRemove', {
+			emitter: 'client',
+			event: 'messageReactionRemove',
+		});
+	}
+
+	async exec(reaction: MessageReaction, user: User) {
+		const msg = reaction.message;
+		if (!msg.guild) return;
+
+		if (user.id === this.client.user?.id) return;
+		if (msg.author.id === this.client.user?.id) return;
+
+		const config = guildConfigs.get(msg.guild.id);
+
+		if (
+			!config ||
+			!config.features.starboard ||
+			!config.features.starboard.channel ||
+			!config.features.starboard.triggerAmount ||
+			(config.features.starboard.ignoredChannels &&
+				config.features.starboard.ignoredChannels.includes(msg.channel.id))
+		)
+			return;
+
+		const addedEmoji = reaction.emoji;
+
+		let triggerEmoji = config.features.starboard.emoji
+			? Starboard.getEmojiFromId(this.client, config.features.starboard.emoji)
+			: '⭐';
+
+		if (!Starboard.emojiEquals(addedEmoji, triggerEmoji)) return;
+
+		const count = reaction.count || 0;
+
+		const channel = msg.guild.channels.cache.get(
+			config.features.starboard.channel
+		) as TextChannel;
+
+		const exists = await Starboard.exists(msg.id);
+		if (!exists) return;
+
+		let entry = await StarModel.findOne({ _id: msg.id });
```

Comment:
* You can make a more descriptive name here, such as `starboardMessage` instead of `entry`. 
* This can be a constant since it's not being altered in the function after declaration.